### ISSUE TITLE
Summerize output

### ISSUE THREAD

**Issue by xamanu** [grote/osm2gtfs#95]
Instead of printing out random messages, it would be nice to get a kind of "report" of the data it ran on, things to improve on OpenStreetMap, etc. These information should be grouped.

**Comment by xamanu** [comment]
Here a quick brainstorming of the things I would like to know:

* Content from the configuration file, also if guessed (like dates)
* How many routes, stops and trips were added.
* How many had errors, warnings
* List elements that need to be fixed on OSM to get into the GTFS
* List elements with recommendations to improve on OSM
* List of queried/guessed stop names

Any more?

**Comment by grote** [comment]
I like the idea, but would like also some way to monitor progress for long-running tasks.

**Comment by xamanu** [comment]
Any suggestion or idea on how to do this, @grote? I would probably open a separate issue fo this. It doesn't seem to be strictly connected and at a first glance much more complex with a need for much more changes than the suggested feature proposed here.

**Comment by grote** [comment]
It is insofar connected as "printing out random messages" currently serves as a progress indicator. If you would remove these, you would also remove the only way to monitor progress. So all I am saying is, please don't remove them without an alternative. This could be as simple as a `--debug` switch.

**Comment by xamanu** [comment]
Yes, I agree. This issue here is about the reports, and should not tackle the removal of any current output messages. In any case if removing them, a suitable way to get feedback on the progress is needed. I opened up a related issue #98 for this.

**Comment by xamanu** [comment]
I made my head a bit more around it and I came to the conclusion that such a summary is probably better to made to analyse whatever GTFS and not just based on `osm2gtfs`.

From my side this can be closed.

**Comment by nlehuby** [comment]
At Jungle Bus, we have some work in progress (quite slow progress to be honest :sweat_smile: ) to have better quality assurance report from OSM transport data with Bifidus project :
* https://jungle-bus.github.io/bifidus/?issues=line_info#13/48.84096/2.38303
* https://github.com/Jungle-Bus/bifidus

For now, mostly focus on explaining the issues and how to fix them and only in French, but a "by network" / "by operator" summary is also planed. Please let me know if you want to work on it with us or if you have special ideas and needs.
